modules/apps/data-engine/data-engine-test/src/testFunctional/macros/DERenderer.macro

```diff
@@ -24,7 +24,6 @@ definition {
 			value1 = "${value}");
 	}
 
-<<<<<<< HEAD
```

Comment:
@guivilarim, this kind of code shouldn't be present in any commit. You should keep it out of you commits. :wink: